src/Spout/Writer/Common/Creator/WriterFactory.php

```diff
@@ -47,7 +47,7 @@ public function create($writerType)
     /**
      * @return CSVWriter
      */
-    private function getCSVWriter()
+    public function getCSVWriter()
```

Comment:
I would not expose these methods. You don't want people using the lib to have access to them.

Reply:
So the duplicated instance create methods would be needed again in Entity Factory?

Reply:
Well I have hidden them again.